loader_hub/google_drive/base.py

```diff
@@ -132,35 +132,61 @@ def _get_fileids_meta(
                     mime_query = " or ".join([f"mimeType='{mime_type}'" for mime_type in mime_types])
                     query += f" and ({mime_query})"
 
-                results = service.files().list(q=query, fields="*").execute()
+                results = service.files().list(
+                    q=query, 
+                    includeItemsFromAllDrives=True,
+                    supportsAllDrives=True, 
+                    fields="*"
+                ).execute()
                 items = results.get("files", [])
                 for item in items:
                     if item["mimeType"] == folder_mime_type:
                         fileids_meta.extend(
                             self._get_fileids_meta(folder_id=item["id"], mime_types=mime_types)
                         )
                     else:
-                        fileids_meta.append(
-                            (
-                                item["id"],
-                                item["owners"][0]["displayName"],
-                                item["name"],
-                                item["createdTime"],
-                                item["modifiedTime"],
+                        # Check if file doesn't belong to a Shared Drive. "owners" doesn't exist in a Shared Drive
+                        if 'driveId' not in item:
+                            fileids_meta.append(
+                                (
+                                    item["id"],
+                                    item["owners"][0]["displayName"],
```

Comment:
instead of having a different length tuple dependent on shared drive or not, can we just have a boolean variable in `fileids_meta` representing `isShared`? 

having a list of tuples that are of varying lengths is not clean, both for typing purposes as well as code processing purposes

Reply:
Good idea. See the change I made